### PR TITLE
Release 0.7.0 Announcement

### PR DESCRIPTION
**Pull request by adambkaplan**
- Initial draft blog post on the announcement from @SaschaSchwarze0 
- My additional updates on the operator release

### CODE REVIEW

File: content/en/blog/posts/2021-12-17-build-release-v0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  - The cosign claims were validated
+  - Existence of the claims in the transparency log was verified offline
+  - Any certificates were verified against the Fulcio roots.
+
+[{"critical":{"identity":{"docker-reference":"ghcr.io/shipwright-io/build/shipwright-build-controller"},"image":{"docker-manifest-digest":"sha256:887b76092d0e6f3c4f4c7b781589f41fde1c967ae9ae62f3a6bdbb18251a562f"},"type":"cosign container image signature"}...
+```
+
+Our signing process takes advantage of the new [keyless mode](https://github.com/sigstore/cosign/blob/main/KEYLESS.md) for cosign and support for [OIDC tokens in GitHub actions](https://chainguard.dev/posts/2021-12-01-zero-friction-keyless-signing).
+
+## shp - New `buildrun delete` Command
```
Comment (gabemontero) [inline content/en/blog/posts/2021-12-17-build-release-v0.7.0.md:32]: this section is a lot more about code scans than the new shp buildrun delete 

I would redo the title along those lines and just lose the shp buildrun delete item

**Comment by adambkaplan** [comment]
/approve

Self-approving

File: content/en/blog/posts/2021-12-17-build-release-v0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+* We worked further on our environment variable support: the Buildpacks sample build strategy now performs the necessary logic to propagate them to the Buildpacks build so that you can customize the behavior of your build using one of the many configurations that the Buildpacks support.
+* We also further enhanced the BuildRun status: for your Git source, we now include the branch name in case you have not specified any revision in your Build. That way you have better insights on what was built.
```
Comment (gabemontero) [inline content/en/blog/posts/2021-12-17-build-release-v0.7.0.md:58]: should we mention the deprecation of implicit emptydir, with the anticipation of ship-22 and build volumes coming in v0.8.0

Comment (gabemontero) [reply content/en/blog/posts/2021-12-17-build-release-v0.7.0.md:58]: I'm fine either way, but thought it worth asking

File: content/en/blog/posts/2021-12-17-build-release-v0.7.0.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+Soon you should be able to see an evolution of parameters: build strategy authors will be able to define array parameters which is required for scenarios such as build-args for Dockerfile-based builds, and Build users will be able to define that the value should be retrieved from a ConfigMap or Secret.
+
+We work on the next evolution of error reporting inside BuildRuns. Where you today just see a BuildRun being failed and must look at the Pod logs to determine the root cause, you will see further error reasons and messages in the BuildRun status directly – with those details provided internally in our Shipwright-managed steps (like the Git step, for example when a revision does not exist), but also by the build strategy author (for example if Buildpacks fail to detect any source code, or a Dockerfile-based strategy when the Dockerfile does not exist).
+
+Finally, we investigated and spiked on different approaches to support building from source code from your local disk rather than from a Git repository. Part of the code is even in the product already, but we need more time to get it done end-to-end.
```
Comment (gabemontero) [inline content/en/blog/posts/2021-12-17-build-release-v0.7.0.md:122]: ok I would definitely mention build volumes here :-)

**Review by gabemontero — CHANGES_REQUESTED** [review]
a couple of minor suggestions

File: content/en/blog/posts/2021-12-17-build-release-v0.7.0.md (hunk 2 above)
Comment (adambkaplan) [reply content/en/blog/posts/2021-12-17-build-release-v0.7.0.md:58]: We should do it here, too. Not everyone reads release notes.

File: content/en/blog/posts/2021-12-17-build-release-v0.7.0.md (hunk 3 above)
Comment (adambkaplan) [reply content/en/blog/posts/2021-12-17-build-release-v0.7.0.md:136]: done in the preceding paragraph

**Review by sbose78 — APPROVED** [review]
Looks good!

**Pull request merged**